metricproxy/exec_proxy.go

```diff
@@ -345,7 +335,7 @@ func (ecp *execCachingProxy) Scrape(ctx context.Context, values url.Values) ([]*
 
 	select {
 	case <-ecp.resultReadyCh:
-		log.Debugln("Returning cached results of scrape")
+		zap.L().Debug("Returning cached results of scrape")
```

Comment:
inconsistency, we do we use global logger here instead of the stored instance?

Reply:
yeas this is a bug, thank you!

Reply:
interesting, this is an upstream-bug, not the one introduced during switch to zap